mythril/analysis/modules/integer.py

```diff
@@ -290,7 +290,7 @@ def _handle_transaction_end(self, state: GlobalState) -> None:
             try:
 
                 transaction_sequence = solver.get_transaction_sequence(
-                    state, node.constraints + [annotation.constraint]
+                    state, state.node.constraints + [annotation.constraint]
```

Comment:
```suggestion
                    state, state.mstate.constraints + [annotation.constraint]
```

Great catch! Probably got in there bc copy pasta.

The suggested change is bc I think we should start to move away from using the constraints stored in the nodes (this is one of those artifacts from the old laser)